app/routes/meetings/publish/publication-actions/detail.js

```diff
@@ -12,8 +12,16 @@ export default class MeetingsPublishPublicationActionsDetailRoute extends Route
       },
     );
     const signedResource = await log.signedResource;
-    if (signedResource) {
+    if (signedResource?.content) {
       return signedResource.content;
+    } else if (signedResource?.file) {
```

Comment:
This condition will always be truthy as long as `signedResource` is defined, as the `file` relationship (event if its content is `null`) will always exist.

Reply:
Good point. I've actually fixed this in my latest changes.